framework/src/postprocessors/ElementExtremeMaterialProperty.C

```diff
@@ -0,0 +1,117 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "ElementExtremeMaterialProperty.h"
+
+#include <algorithm>
+#include <limits>
+
+registerMooseObject("MooseApp", ElementExtremeMaterialProperty);
+
+defineLegacyParams(ElementExtremeMaterialProperty);
```

Comment:
Oh, I'm not an expert here, but if we can get away without defining this, let's avoid it.